scripts/fetch_ext_bins.sh

```diff
@@ -90,4 +90,5 @@ function fetch_tools {
 
   mkdir -p "${tools_bin}"
   mv "${kb_root_dir}"/bin/* "${tools_bin}"
+  rm -f "${tools_bin}"/bin/kubectl
```

Comment:
why do we need this?

Reply:
Yes. Without doing this the CI jobs won't run locally due to us using an older version of kubectl than required by the CI scripts (1.16.4 vs 1.16.7+). I tried to update the kubebuilder artifacts, but it appears that artifacts for 1.16.4+ are not available.

Based on the above, I thought it best to just download kubectl separately.